boolean/reverse condition/4aca7b99-e216-476e-8e91-6d7d172fca16.json

```diff
@@ -0,0 +1,41 @@
+{
+  "implementations": {
+    "es6-node": {
+      "inputFunctions": {
+        "e7417484-ea01-4448-9666-f601edcaac11": "outputs.boolean(true, tag);",
+        "883c5c6b-7512-481d-b7b7-0f988046cc76": "outputs.boolean(false, tag);"
+      }
+    }
+  },
+  "interface": {
+    "inputs": {
+      "e7417484-ea01-4448-9666-f601edcaac11": {
+        "type": "any",
+        "name": "true",
+        "description": "Receives signals to be evaluated to true."
+      },
+      "883c5c6b-7512-481d-b7b7-0f988046cc76": {
+        "type": "any",
+        "name": "false",
+        "description": "Receives signals to be evaluated to false."
+      }
+    },
+    "outputs": {
+      "9c95f7fd-ee7e-41fb-be63-361f33159a77": {
+        "type": "boolean",
+        "name": "boolean",
+        "description": "Sends boolean depending on which input the signal was received through."
+      }
+    }
+  },
+  "name": "boolean/Reverse condition",
+  "attributes": {
+    "locked": true
+  },
+  "description": "Turns binary independent signals into a boolean.\n\nExample A:\n1. null@0 is received via `true`\n2. true@0 is sent via `boolean`\n\nExample B:\n1. \"foo\"@1 is received via `false`\n2. false@1 is sent via `boolean`",
```

Comment:
I think we already have a prototype for this purpose: `boolean/To boolean`

Reply:
Forgot about it. I'll remove the new one, but adjust names / description on the existing one.